### ISSUE TITLE
Back/Forward Navigations Focus Pane but Require Multiple Clicks to Navigate

### ISSUE THREAD

**Issue by rivahere** [files-community/Files#15706]
### Description

When in dual pane mode, when I hover over the unfocused pane and use my back/forward buttons on my mouse, it will highlight the pane as if it is now in focus but still navigates the original now un-highlighted pane until I click the highlighted pane or hit back/forward a second or third time.

### Steps To Reproduce

1.) Open dual pane mode.
2.) Use hardware forward/backward navigation keys on the pane opposite to the one in focus.

### Requirements

Complete the navigation on the pane the mouse pointer is hovered over on the first click.

### Files Version

3.5.0.0

### Windows Version

Windows 11 Pro 22621.3810

### Log File

[debug.log](https://github.com/user-attachments/files/15994211/debug.log)

**Comment by yaira2** [comment]
Is this request that clicking on the mouse back button should focus the hovered pane?

**Comment by rivahere** [comment]
> Is this request that clicking on the mouse back button should focus the hovered pane?

yes, instead of having to click it twice or the primary mouse button. As it is now, it highlights as if it focuses it but isn't actually focused.

**Comment by yaira2** [comment]
Navigating back can be confusing if it's not the selected pane, but we should definitely fix the issue where the pane is highlighted.

**Comment by hishitetsu** [comment]
I found that pressing the back/forward button on the unfocused pane will highlight and focus that pane. This behavior is same as Edge.
Should this behavior still be changed?

**Comment by yaira2** [comment]
It's okay with me then, if the standard practice is to focus the hovered pane, the main thing is to be consistent.

**Comment by yair100** [comment]
@rivahere can you confirm if this is still an issue in the latest preview build (v3.9.15)? https://files.community/download

**Comment by Josh65-2201** [comment]
@rivahere Can you answer the above question

**Comment by yair100** [comment]
@Josh65-2201 were you able to reproduce this?

**Comment by Josh65-2201** [comment]
Not having the issue of the other pane moving forward but highlighing the hovered one in v4. It focus the one I'm hoving and move that one forward and other control happen in the focused one.
